src/viewer.js

```diff
@@ -153,14 +175,7 @@ async function generateReport(bundleStats, opts) {
             return;
           }
 
-          const reportFilepath = path.resolve(bundleDir || process.cwd(), reportFilename);
```

Comment:
Seems like there were few possible cases:

1. Manually pass in report name to Plugin config/CLI options, then the cli/plugin resolves the full path before passing it to the `generateReport`, and `path.resolve(bundleDir || process.cwd(), reportFilename);` would always simply return `reportFilename`.

2. You do not pass report name to Plugin config/CLI options. They default to `report.html` and the path is also resolved before calling `generateReport`. Same happens as in case 1.

So overall path.resolve and || inside `generateReport` were redundant. Unless I am missing something :)

Since I am splitting out json/html generation to two separate functions I will revert back path resolvement to outside of `generateReport`, to make sure that I am not changing existing functionality.

Reply:
Thanks! Seems like indeed it might be redundant, but better play it safe and not change it here :smile: